.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ on:
     paths-ignore:
       - "**/*.md"
 
-permissions: read
+permissions: read-only
```

Comment:
Maybe i am looking at the wrong spot but i dont see a `read-only` permission https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token

Reply:
bleerrrrrg. you right. gonna revert this and see what the previous error was.

Reply:
```suggestion
permissions: read
```

Reply:
fixed